### PR TITLE
Linsun/convint8 fwd instances

### PR DESCRIPTION
**Pull request by linsun12**
This pull request is for adding int8 support for 2d grouped convolution fwd on gfx9 architecture using xdlops. The following instances are added:

For layout GNHWC/GKYXC/GNHWK
grouped_conv2d_fwd_xdl_gnhwc_gkyxc_gnhwk_int8_instances

For layout NHWGC/GKYXC/NHWGK
grouped_conv2d_fwd_xdl_nhwgc_gkyxc_nhwgk_int8_instances
grouped_conv2d_fwd_xdl_large_tensor_nhwgc_gkyxc_nhwgk_int8_instances
grouped_conv2d_fwd_xdl_merged_groups_nhwgc_gkyxc_nhwgk_int8_instances
grouped_conv2d_fwd_xdl_nhwgc_gkyxc_nhwgk_int8_comp_instances
grouped_conv2d_fwd_xdl_nhwgc_gkyxc_nhwgk_int8_mem_intra_instances
grouped_conv2d_fwd_xdl_nhwgc_gkyxc_nhwgk_int8_mem_inter_instances

For layout NGCHW/GKYXC/NGKHW
grouped_conv2d_fwd_xdl_nhwgc_gkyxc_nhwgk_int8_instances
grouped_conv2d_fwd_xdl_merged_groups_ngchw_gkyxc_ngkhw_int8_instances
grouped_conv2d_fwd_xdl_ngchw_gkyxc_ngkhw_int8_comp_instances
grouped_conv2d_fwd_xdl_ngchw_gkyxc_ngkhw_int8_mem_intra_instances
grouped_conv2d_fwd_xdl_ngchw_gkyxc_ngkhw_int8_mem_inter_instances

### CODE REVIEW

File: library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp [hunk 1]
```diff
@@ -189,9 +192,17 @@ struct DeviceOperationInstanceFactory<ck::tensor_operation::device::DeviceGroupe
             {
                 add_device_grouped_conv2d_fwd_xdl_gnhwc_gkyxc_gnhwk_bf16_instances(op_ptrs);
             }
+#endif
+#ifdef CK_ENABLE_INT8
+            if constexpr(is_same_v<InDataType, int8_t> && is_same_v<WeiDataType, int8_t> &&
+                         is_same_v<OutDataType, int8_t> && is_same_v<AComputeType, int8_t> &&
+                         is_same_v<BComputeType, int8_t>)
+            {
+                add_device_grouped_conv2d_fwd_xdl_gnhwc_gkyxc_gnhwk_int8_instances(op_ptrs);
```
Comment (bartekxk) [inline library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp:201]: Do we need instances in gnhwc layout?

File: profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp [hunk 2]
```diff
@@ -202,7 +203,8 @@ bool profile_grouped_conv_fwd_impl(int do_verification,
         }
         else
         {
-            std::cout << op_ptr->GetTypeString() << " does not support this problem" << std::endl;
+            // std::cout << op_ptr->GetTypeString() << " does not support this problem" <<
+            // std::endl;
```
Comment (bartekxk) [inline profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp:207]: should be restored

File: profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp [hunk 3]
```diff
@@ -155,6 +155,7 @@ bool profile_grouped_conv_fwd_impl(int do_verification,
 
         if(op_ptr->IsSupportedArgument(argument_ptr.get()))
         {
+            std::cout << op_ptr->GetTypeString() << " supports this problem" << std::endl;
```
Comment (bartekxk) [inline profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp:158]: I suppose it is not needed. We print perf for supported instance

File: test/grouped_convnd_fwd/test_grouped_convnd_fwd.cpp [hunk 4]
```diff
@@ -41,7 +41,7 @@ class TestGroupedConvndFwd : public ::testing::Test
                                true,  // do_verification
                                1,     // init_method: integer value
                                false, // do_log
-                               false, // time_kernel
+                               true,  // time_kernel
```
Comment (bartekxk) [inline test/grouped_convnd_fwd/test_grouped_convnd_fwd.cpp:44]: by defualt should be disabled in tests

File: library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp (hunk 1 above)
Comment (linsun12) [reply library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp:201]: This is following fp16's pattern, can be deleted.

File: profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp (hunk 2 above)
Comment (linsun12) [reply profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp:207]: Will delete

File: profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp (hunk 3 above)
Comment (linsun12) [reply profiler/include/profiler/profile_grouped_conv_fwd_impl.hpp:158]: will delete

File: test/grouped_convnd_fwd/test_grouped_convnd_fwd.cpp (hunk 4 above)
Comment (linsun12) [reply test/grouped_convnd_fwd/test_grouped_convnd_fwd.cpp:44]: will disable timer

**Pull request closed without merging**